lib/nunjucks/govuk-prototype-kit/internal/views/manage-prototype/plugin-install-or-uninstall.njk

```diff
@@ -15,6 +15,9 @@
       <p>
         In terminal, run:</p>
       <p><code>{{ command }}</code></p>
+      <p>
+        When you've installed the plug-in, restart your prototype in the terminal by typing: npx govuk-prototype-kit start</p>
```

Comment:
Does the user need to stop the kit before running the install command? Right now it isn't clear to me from this text.

Reply:
I did mention pressing ctrl-c but I think @NoraGDS thought the above was clear enough.

Reply:
Okay